include/bswap.h

```diff
@@ -42,6 +45,18 @@
 #endif
 #endif
 
+/* macOS */
+#ifdef __APPLE__
+#define __LITTLE_ENDIAN LITTLE_ENDIAN
+#define __BIG_ENDIAN BIG_ENDIAN
+#define __BYTE_ORDER BYTE_ORDER
+#if __BYTE_ORDER == BIG_ENDIAN
+#define __BIG_ENDIAN_BITFIELD
+#elif __BYTE_ORDER == LITTLE_ENDIAN
```

Comment:
With respect to pali#14, this needs to be sorted out in same way to prevent future conflict.